### PR TITLE
Beta 5.0.1 RC2

### PR DESCRIPTION
**Pull request by tmolitor-stud-tu**
- Show Debug menu after tapping >= 16 times onto app version (stable only, beta will always show it)
- Don't drop file download errors silently
- Don't log outgoing sasl and passwrd change stanzas (your password won't be logged anymore)
- Trim whitespaces and newlines at the beginning or end of a message
- Fix microphone icon not showing always
- Renamed "Log" to "Debug" in settings menu
- Move contact details close button to the left
- Fix some very rare TCP stream handling bugs.
- Fix old xmpp resources created with monal < 4.3 not having a random part

### CODE REVIEW

**Review by FriedrichAltheide — DISMISSED** [review]

File: .github/ISSUE_TEMPLATE/custom.md [hunk 1]
```diff
@@ -12,17 +12,20 @@ assignees: ''
...
 ### Additional information like screenshots:
 
 
-If you can please activate the logs via `Settings/Log` - then reproduce your issue and then send the file to info (at) monal (dot) im.
+If you can reproduce your issue: send us your log file (`Settings/Log`) to info (at) monal (dot) im.
 PLEASE BE AWARE THAT THIS CAN PROVIDE SENSITIVE INFORMATION TO THE DEVELOPERS (which of course will not abuse this, but you are informed now!)
 
 ---
-[ ] I have cross-check this overview https://github.com/anurodhp/Monal/issues/322 as well es filtered for related labels https://github.com/anurodhp/Monal/labels
+[ ] I have cross-check this overview https://github.com/monal-im/Monal/issues/322 as well es filtered for related labels https://github.com/anurodhp/Monal/labels
```
Comment (janst97) [inline .github/ISSUE_TEMPLATE/custom.md:29]: I have cross-check**ed** this overview https://github.com/monal-im/Monal/issues/322 as well **as** filtered for related labels https://github.com/anurodhp/Monal/labels

File: .github/ISSUE_TEMPLATE/custom.md [hunk 2]
```diff
@@ -12,17 +12,20 @@ assignees: ''
...
+[ ] I have checked that at least XEP-198, XEP-0280, XEP-0352, XEP-0357 and XEP-0163 are activated on your server and showed as 'green' under Settings-->Account-->(i) in server column (under advanced settings)
```
Comment (janst97) [inline .github/ISSUE_TEMPLATE/custom.md:30]: [ ] I have checked that at least XEP-198, XEP-0280, XEP-0352, XEP-0357 and XEP-0163 are activated on my server and displayed as 'green' under Settings-->Account-->(i) in server column (under advanced settings)

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+Please report issues directly to info@monal.im. Feel free to put in CC: thilo+molitor@eightysoft.de
+
+Please try to report 
+- detailed what you are concerned about
+- if applicable, how to reproduce
+- your contact details, if the sending email is not enough. That way we can ask questions back to you. 
+
+You are also invited to make a recommendation on how to fix a potential security vulnerability.
+
+Once a vulnerability has been announced and indicated we try our very best to provide a correct as soon as possible,
```
Comment (janst97) [inline SECURITY.md:17]: to provide a fix

File: SECURITY.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+at its best within days. However, dependent on the potential issue it can take longer if many code sections needs to be changed.
```
Comment (janst97) [inline SECURITY.md:18]: code sections need to be changed.

File: SECURITY.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+Please remind that this is a non-commercial software project.
```
Comment (janst97) [inline SECURITY.md:19]: Please be reminded

File: SECURITY.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+It is highly appreciated to report a vulnerability to the Monal developers. We ask you for
+disclosure until it has been fixed. The prevents abuse and exploitation in the current published releases.
```
Comment (janst97) [inline SECURITY.md:6]: This prevents abuse

File: SECURITY.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+
+Please report issues directly to info@monal.im. Feel free to put in CC: thilo+molitor@eightysoft.de
+
+Please try to report 
+- detailed what you are concerned about
```
Comment (janst97) [inline SECURITY.md:11]: - in detail what you are concerned about

**Pull request merged**